packages/next-templates/src/app/meldingen/verhuizing-doorgeven/page.tsx

```diff
@@ -0,0 +1,398 @@
+'use client';
+
+import {
+  Article,
+  AccordionProvider,
+  ButtonLink,
+  ButtonGroup,
+  Document,
+  Heading1,
+  Heading2,
+  Heading3,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+import '@/app/styling/css/detail-page.css';
+import '@utrecht/design-tokens/dist/index.css';
+import { ExampleHeaderFunnel } from '@/components/ExampleHeader/ExampleHeaderFunnel/ExampleHeaderFunnel';
+import { ExampleFooterFocus } from '@/components/ExampleFooter/ExampleFooterFocus/ExampleFooterFocus';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { UtrechtDigidLogo, UtrechtIconArrow } from '@utrecht/web-component-library-react';
+import { useForm } from 'react-hook-form';
+import { useRouter } from 'next/navigation';
+import BacklinkIcon from '@/app/styling/assets/backlink-icon.svg';
+
+export default function Home() {
+  const router = useRouter();
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<{ [key: string]: string }>();
+
+  return (
+    <Document>
+      <Page>
+        <ExampleHeaderFunnel />
+        <ExampleNavigation />
+        <PageContent>
+          <Article className="voorbeeld-article-space">
+            <Link className="voorbeeld-link-back" href="./">
+              <BacklinkIcon />
+              <span className="voorbeeld-span--underlined">Terug</span>
```

Comment:
Kunnen we deze code versimpelen dat we geen `span` en geen extra class name nodig hebben?